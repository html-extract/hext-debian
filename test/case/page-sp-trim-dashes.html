<a> - - - - - Word - - - - - - </a>
<a>-Word-</a>
<a>- Word -</a>
