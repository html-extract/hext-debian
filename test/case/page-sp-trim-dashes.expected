{"title":"Word"}
{"title":"Word"}
{"title":"Word"}
